release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -519,6 +519,12 @@ Open Virtual Network (OVN) was redesigned to host its control plane and data sto
 
 For clusters on user-provisioned xref:../installing/installing_bare_metal/installing-bare-metal-network-customizations.adoc#installation-configuration-parameters-network_installing-bare-metal-network-customizations[bare metal infrastructure], the OVN-Kubernetes cluster network provider supports both IPv4 and IPv6 address families.
 
+[id="ocp-4-11-pod-level-bonding"]
+==== Pod-level bonding for secondary networks
+Bonding at the pod level is vital to assist workloads inside pods that require high availability and more throughput. With pod-level bonding, you can create a bond interface from many single root I/O virtualization (SR-IOV) virtual function interfaces in kernel mode interface. The SR-IOV virtual functions are passed into the pod and attached to a kernel driver. This feature was previously introduced as a Technology Preview feature in {product-title} 4.10 and is now generally available in {product-title} 4.11. 
```

Comment:
The 4.10 and 4.11 backports work is still in progress. The release note I suggested was for 4.12. Until the development and testing is completed for 4.11, we should not release note for 4.11.